src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -690,6 +703,61 @@ private static bool IsNodeOrAnyAncestorLeftSideOfDot(SyntaxNode node, ISyntaxFac
 
             return false;
         }
+
+        internal static ImmutableArray<FindUsageProperty> GetAdditionalProperties(SyntaxNode node, SemanticModel semanticModel, ISyntaxFactsService syntaxFacts)
+        {
+            var additionalProperties = new ArrayBuilder<FindUsageProperty>
```

Comment:
I'm curious if this change would have any impact on perf. Have you done any profiling? 
@cyrus.najmabadi@gmail.com I vaguely remember seeing some sort of symbol cache for FAR, is that correct?